websocket/src/index.ts

```diff
@@ -1,20 +1,85 @@
+import express from "express";
+import http from "http";
+import cors from "cors";
 import { Server } from "socket.io";
 
-const io = new Server({
-    cors: {
-    origin: ["*"]
-    }
+const app = express();
+const server = http.createServer(app);
+const io = new Server(server, {
+  cors: {
+    origin: "*",
```

Comment:
at very later stage, remember to allow only known hosts/origings,

Reply:
sure